portfolio/src/main/webapp/index.html

```diff
@@ -24,12 +24,17 @@ <h3>Photos</h3>
         <div class="text">
           <p>This is my portfolio.</p>
           <p>I am a student at Imperial College London, and this is my first
-             project as an intern at Google! This is a small portfolio of 
+              project as an intern at Google! This is a small portfolio of 
              myself and my interests, as well as links to where you can see 
-             more of my work and projects. Enjoy!</p>
+             more of my work and projects.</p>
+          <p>I have recently gotten more into photography, so I have dedicated 
+              a separate page to highlight some of my favourite photos that 
+              I've taken (mostly on my Pixel 2xl).
+          <p>Enjoy browsing my page!</p>
         </div>
         <a href="/images/me_main.jpeg"><img src="/images/me_main.jpeg"/></a>
       </div>
+      <button href="/pics.html">heck out my favourite photos!</button>
```

Comment:
This button doesn't work.

In general, it's fine to change little things after review, but some bigger features like a new pic page should be reviewed separately

Reply:
Also, heck, you have a typo here :)